osf/models/schema_response.py

```diff
@@ -399,7 +400,7 @@ def _validate_accept_trigger(self, user):
             # not self.pending_approvers.exists() -> called from within "approve"
             if user is None or not self.pending_approvers.exists():
                 return
-            raise ValueError(
```

Comment:
I assume this should throw as a 409.

Reply:
Not technically a machine error, but work for me